app/model/Notification.php

```
@@ -39,18 +42,26 @@ public function __construct(
+                    $link = $this->linkGenerator->link('Challenge:default');
+                    break;
+                case self::MESSAGE_NEW_FRIEND_REQUEST:
+                    $link = $this->linkGenerator->link('Friends:default');
+                    break;
+                case self::MESSAGE_NEW_GROUP_INVITATION:
+                    $link = $this->linkGenerator->link('Group:default');
+                    break;
+                default:
+                    $link = null;
+            }
         }
```

PHPMD (2.15.0)

[error] The method insertNotification uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)